operations/paymentrequests.md

```diff
@@ -1,306 +1,312 @@
-            "ReservationId": "9e6d4492-315b-4089-b9d6-5b1bd2eddc1b",
-            "Notes": "Internal notes."
-        }
-    ]
-}
-``` 
-
-| Property | Type | Contract | Description |
-| :-- | :-- | :-- | :-- |
-| `PaymentRequests` | array of [Payment requests](#payment-request) | required | Added payment requests. |
-
-## Cancel payment requests
-
-Cancels specified payment requests. Only payment requests which are in `Pending` state can be canceled.
-
-### Request
-
-`[PlatformAddress]/api/connector/v1/paymentRequests/cancel`
-
-```javascript
-{
-    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
-    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
-    "Client": "Sample Client 1.0.0",
-    "PaymentRequestIds": [
-        "6282d17b-a068-4a9f-83d3-afae00c39bfb"
-    ]
-}
-```
-
-| Property | Type | Contract | Description |
-| :-- | :-- | :-- | :-- |
-| `ClientToken` | string | required | Token identifying the client application. |
-| `AccessToken` | string | required | Access token of the client application. |
-| `Client` | string | required | Name and version of the client application. |
-| `PaymentRequestIds` | array of string | required, max 1000 items | Identifiers of payment requests to be canceled. |
-
-### Response
-
-```javascript
-{
-    "PaymentRequests": [
-        {
-            "Id": "6282d17b-a068-4a9f-83d3-afae00c39bfb",
-            "AccountId": "8466dfdd-0964-4002-8719-afa900d0f1ba",
-            "CreatedUtc": "2023-10-01T11:48:57Z",
-            "UpdatedUtc": "2023-10-28T11:48:57Z",
-            "ReservationGroupId": null,
-            "State": "Canceled",
-            "Amount": {
-                "Currency": "EUR",
-                "NetValue": 10,
-                "GrossValue": 10,
-                "TaxValues": [],
-                "Breakdown": {
-                    "Items": [
-                        {
-                            "TaxRateCode": null,
-                            "NetValue": 10,
-                            "TaxValue": 0
-                        }
-                    ]
-                }
-            },
-            "Type": "Payment",
-            "Reason": "PaymentCardMissing",
-            "ExpirationUtc": "2023-02-20T12:00:00Z",
-            "Description": "Payment required",
-            "ReservationId": "9e6d4492-315b-4089-b9d6-5b1bd2eddc1b",
-            "Notes": "Internal notes."
-        }
-    ]
-}
-``` 
-
-| Property | Type | Contract | Description |
-| :-- | :-- | :-- | :-- |
-| `PaymentRequests` | array of [Payment requests](#payment-request) | required | Canceled payment requests. |
+<!-- AUTOMATICALLY GENERATED, DO NOT MODIFY -->
+# Payment requests
+
+## Get all payment requests
+
+Get all payment requests belonging to the specified customer accounts. 
+Note this operation uses [Pagination](../guidelines/pagination.md) and supports [Portfolio Access Tokens](../concepts/multi-property.md).
```

Comment:
```suggestion
Get all payment requests belonging to the specified customer accounts. Note this operation uses [Pagination](../guidelines/pagination.md) and supports [Portfolio Access Tokens](../concepts/multi-property.md).
```
nitpick: It should not be on a new line

Reply:
![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExb3ZrcXF0YXNveGtreXk4bWM2Y2VzN3Y5cTg2MnAwYjloNGZjenRwaiZlcD12MV9naWZzX3NlYXJjaCZjdD1n/LtVqGoxh5v9DSpwBVQ/giphy.gif)